tasks/Mock Jury.md

```diff
@@ -0,0 +1,42 @@
+# Summary
+The experiment performed was designed to test the effects of group deliberation on jurors when exposed to inadmissable evidence.
+Speciffically the study sought to see how group deliberation would effect jurors individual ability to disregard inadmissable evidence.
+
+# References
+London, K., & Nunez, N. (2000). The effect of jury deliberations on jurors' propensity to disregard inadmissible evidence. Journal of Applied Psychology, 85(6), 932–939. https://doi.org/10.1037/0021-9010.85.6.932
+
+Link: https://psycnet-apa-org.proxy.library.upenn.edu/record/2000-16508-010
+
+# Stimuli
+## The visual components
+N/A 
+
+## Materials for alternative versions of the experiment 
+3 versions of a mock trial script.  All versions of the script should contain testimones from both the defense and the prosecution.  The only difference in the versions of the scipts are the extra admissable/inadmissable evidence. The three versions should be as follows:
+- Control group script, containing a script from a mock trial
+- Admissable group script, the same script as the control group, containing an extra piece of evidence that the group is allowed to use in their decision making
+- Inadmissable group script, the same script as the control group, containing the same extra piece of evidence as the Admissable group, however, this group is told the extra evidence is inadmissable
+
+# Procedure
+## Steps
+All participants will act as jurors.
+1) Jurors are split into groups ranging in size of 8-12. (Average group size should be around 10)
+2) The groups are presented with a script from a mock trial.  
+  - Some of the groups are given the "regular" trial, with no extra admissable or inadmissable evidence, these are the control groups.
+  - Other groups are presented are given a version of the trial that has extra admissable evidence.
+  - The last groups are given a version of the trail with the same extra evidence, but they are told is is inadmissable and must be ignored, but not told why it is inadmissable.
+3) Each person is to read the trial and act as a juror, they then fill out a pre-deliberation questionaire, where they are asked their personal verdict(guilty or not guilty) and their confidence in their decision(from0 to 100%).  They then deliberate with their group and the goal is to reach a unanimous verdict, if they cannot rreach this within an hour they are cut off.  They then fill out a post-deliberation questionaire where they are once again asked about their individual verdict and confidence in their decision.
+
+## Roles 
+Jurors
+- All participants act as jurors on a regular jury
+
+## Instructions
+All groups are to act as juries.  Juries with the inadmissable evidence are supposed to not let that information bias them.
+
+# Criteria
+## Performance calculation
+Not addressed within the paper, the focus of the experiment was to test the effects of group deliberation on each jurors specific bias stemming from inadmissable evidence.
```

Comment:
I would add that participants filled out a questionnaire: 
Perceptions of Guiltt: Participants filled in a questionnaire regarding their perceptions of the defendant's guilt. Mock jurors rendered a verdict (guilty or not guilty). 

Guilt Score:  9-point scale ranging from 1 (definitely not guilty) to 9 (definitely guilty) the degree to which they felt the defendant was culpable.
 > The degree of guilt score allowed comparisons of the degree tow hich participants felt the defendant was culpable, regardless of whether they decided to vote guilty or not guilty. 

Confidence Scale: All participants indicated on a 9-point scale ranging from 1 (not at all) to 9 (very) the degree of confidence they held in their verdict. 

Culpability Score: Jurors in the inadmissible condition also indicated on a 9-point scale a perceived culpability score (their predicted estimate of how guilty they would have viewed the defendant to be had the critical evidence not been presented).

Reply:
Since participants had a time limit would performance also look at if they reached a decision within that time limit?

Reply:
I will ask about this in the email